engine/apps/api/serializers/user.py

```diff
@@ -31,6 +31,25 @@ class UserPermissionSerializer(serializers.Serializer):
     action = serializers.CharField(read_only=True)
 
 
+class GoogleCalendarSettingsSerializer(serializers.Serializer):
+    # # TODO: figure out how to get OrganizationFilteredPrimaryKeyRelatedField to work with many=True
+    # specific_oncall_schedules_to_sync =
+    # specific_oncall_schedules_to_sync = serializers.ListField(
+    #     child=OrganizationFilteredPrimaryKeyRelatedField(
+    #         queryset=OnCallSchedule.objects,
+    #         required=False,
+    #         allow_null=True,
+    #     ),
+    #     required=False,
+    #     allow_null=True,
+    # )
+    specific_oncall_schedules_to_sync = serializers.ListField(
```

Comment:
This isn't used atm, right?

Reply:
It is used. The UI supports letting you specify these schedules
![Screenshot 2024-03-28 at 14 27 55](https://github.com/grafana/oncall/assets/9406895/54b438a9-8778-45ee-91ea-57d1e91098b7)

Plus it's used [here](https://github.com/grafana/oncall/pull/4104/files#diff-10490f79ae43ef85601c4ec568c12de21e185e70bbbdcc027a6a26d8f6d1ed71R25-R29) in the celery task (next PR)